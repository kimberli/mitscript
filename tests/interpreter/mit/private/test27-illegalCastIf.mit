x = 5;
if (x == 5){
	print("This should print");
}

if (x + 5){
	print("This should've been an illegal cast exception");
}
