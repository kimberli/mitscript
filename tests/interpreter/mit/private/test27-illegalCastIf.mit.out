This should print
IllegalCastException: Uninitialized to Bool
